smurfs/src/Functions/Form.js

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
```

Comment:
add a bunch of comments, aka comment what you are doing for parts that are easy, and for things that are not so easy comment what you want to do